src/components/container/toDoList/toDo.js

```diff
@@ -1,24 +1,26 @@
-import React, { Component, PropTypes } from 'react'
-import { connect } from 'react-redux'
-import { deleteToDo } from '../../../actionCreators/toDoList_action_creator'
-import { toggleToDo } from '../../../actionCreators/toDo_action_creator'
-import ToDo from '../../presentation/toDoList/toDo'
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { deleteToDo } from '../../../actionCreators/toDoList_action_creator';
+import { toggleToDo } from '../../../actionCreators/toDo_action_creator';
+import ToDo from '../../presentation/toDoList/toDo';
 
 class ToDoContainer extends Component{
   _onToggle = () => {
+    //Add toggle to propTypes
```

Comment:
Yeah so this has only come about for because I updated my solution to use ESLint. One of the warnings is anything that is a prop should be declared as in propTypes. It makes it a hell of a lot easier when a project gets big and something isnt working correctly to track the issue down